package.json

```diff
@@ -7,7 +7,7 @@
     "dev": "webpack -d --watch --color --progress",
     "build": "cross-env NODE_ENV=production webpack -p",
     "lint": "./node_modules/.bin/eslint src/client/app/**/*.js src/client/app/**/*.jsx src/server/**/*.js src/server/**/*.jsx src/scripts/**/*.js",
-    "test": "./node_modules/.bin/mocha \"src/server/test/**/*.js\"",
+    "test": "./src/scripts/checkheader.sh && ./node_modules/.bin/mocha \"src/server/test/**/*.js\"",
```

Comment:
Doesn't this make tests not run if `checkheader` fails? We probably want to run both, right?